README.md

```diff
@@ -11,21 +11,21 @@ pip install pbiapi
 Add the client to your project with:
 
 ```python
-from pbiapi import PowerBiApiClient
+from pbiapi import PowerBIAPIClient
 ```
 
-Initiate the client by running: 
+Initiate the client by running:
 ```python
-pbi_client = PowerBiApiClient( 
+pbi_client = PowerBIAPIClient(
     <Tenant Id>,
     <Application Id>,
-    <Service Principal Secret>
-    )
+    <Service Principal Secret>,
+)
 ```
 
 You can then get all the workspaces the Service Principal is admin of by running:
 ```python
-pbi_client.getWorkspaces()
+pbi_client.get_workspaces()
+# Or access attribute directly:
 pbi_client.workspaces
```

Comment:
Should this still be possible? Are we storing the state in the client?

Reply:
Currently yes --> I'm in favour of removing any state/cache variables like workspaces.